wagtail_localize/modeladmin/templates/wagtail_localize/modeladmin/translatable_edit.html

```diff
@@ -7,7 +7,7 @@
 
 {% block form_actions %}
   <div class="dropdown dropup dropdown-button match-width">
-    <button type="submit" class="button action-save button-longrunning" data-clicked-text="{% trans 'Saving…' %}">
+    <button type="submit" class="button action-save button-longrunning" data-controller="w-progress" data-action="w-progress#activate" data-w-progress-active-value="{% trans 'Saving…' %}">
```

Comment:
have you tested this in Wagtail 4.1? the `data-controller` change only happens in Wagtail 5.0